worlds/psychonauts/test/test_goal_options.py

```diff
@@ -0,0 +1,160 @@
+from typing import ClassVar, Iterable
+import random
```

Comment:
No, it's used as part of test class setup, before any world exists.

I guess `victory_items` could be replaced with a `get_victory_items()` method that takes a `world` argument so that `world.random` can be used.

Reply:
If you want your tests to be reproducible by seed in case they error, that sounds like it would be a good idea.